boxing/src/gym/Boxer.java

```diff
@@ -0,0 +1,72 @@
+package gym;
+
+import java.util.Objects;
+
+public class Boxer{
+    private String name;
+    private int age;
+    private int hitPoints;
+    private int power;
+    private Integer wins;
```

Comment:
do we need this to be a reference type? with this, you are explicitly allowing it to be null. if you want to use this behaviour, it is fine, otherwise, a primitive int is enough